pyriemann/utils/kernel.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from .mean import mean_riemann
+from .base import invsqrtm, logm
+
+
+def kernel_riemann(X, Y=None, Cref=None):
```

Comment:
this is to me ambiguous as you can use many metrics. Here one is implied

Reply:
this is the way it's been done in this package, e.g. in distance, geodesic. they're called distance_riemann, geodesic_riemann and so on, fo other metrics they're called distance_logeuclid, geodesic_logeuclid etc. the generic thing is then called distance or geodesic with a string passed. I can implement the generic thing with a string passed as well (although it takes only one string so far).

Reply:
@qbarthelemy what do you think?

Reply:
I added the generic kernel function (which obviously doesn't do anything but the riemannian kernel) to conform with the previous style